geospaas_harvesting/harvesters.py

```diff
@@ -17,13 +17,20 @@ class HarvesterConfigurationError(Exception):
     missing mandatory sections
     """
 
+
 class Harvester():
     """
     Base Harvester class. Implements the basic behavior but is not meant to be used directly.
     It should be subclassed, and child classes should implement the following methods:
         - _create_crawlers()
         - _create_ingester()
+
+    Child classes should also assign values to the ingester and crawler class attributes.
+    These values should be the crawler and ingester classes to be used in the _create_crawlers()
+    and _create_ingester() methods.
     """
+    ingester = None
+    crawler = None
```

Comment:
These should be moved to the `WebDirectoryHarvester` class, because not all harvesters use them

Reply:
Done in the next commit